po/es.po

```diff
@@ -216,30 +167,431 @@ msgstr "_Acerca de Eyedropper"
+msgstr "Dividir-complementario"
+
+#: src/widgets/palette_dialog.rs:154
+msgctxt ""
+"Name of the color palette, which would form a triangle above the color wheel"
+msgid "Triadic"
+msgstr "Triádico"
+
+#: src/widgets/palette_dialog.rs:159
+msgctxt "The name of the color palette."
+msgid "Tetradic"
+msgstr "Tetrádica"
+
+#: src/widgets/palette_dialog.rs:165
+msgctxt "Color palette consisting of six slight shifted colors"
+msgid "Analogous"
+msgstr "Análogo"
+
+#: src/widgets/preferences.rs:147
+msgctxt ""
+"Name of the basic color keyword set from https://www.w3.org/TR/css-color-3/"
+"#html4"
+msgid "Basic"
+msgstr "Básico"
+
+#: src/widgets/preferences.rs:149
+msgid "Show color names from the w3c basic color keyword set"
+msgstr "Mostrar los nombres de los colores del conjunto de palabras clave de color básico w3c"
+
+#: src/widgets/preferences.rs:155
+msgctxt ""
+"Name of the extended color keyword set from https://www.w3.org/TR/css-"
+"color-3/#svg-color"
+msgid "Extended"
+msgstr "Extendido"
+
+#: src/widgets/preferences.rs:157
+msgid "Show color names from the w3c extended color keyword  set"
+msgstr "Mostrar los nombres de los colores del conjunto de palabras clave de color ampliado de la w3c"
+
+#: src/widgets/preferences.rs:161
+msgctxt "Name of the color set from the xkcd color survey"
+msgid "xkcd"
+msgstr "xkcd"
+
+#: src/widgets/preferences.rs:162
+msgid "Show color names from the xkcd color survey"
+msgstr "Mostrar nombres de colores de la encuesta de colores de xkcd"
+
+
+#: src/widgets/preferences.rs:301
 msgid "Move Up"
 msgstr "Subir de categoría"
 
-#: src/widgets/preferences.rs:216
+#: src/widgets/preferences.rs:303
 msgid "Move Down"
 msgstr "Bajar de categoría"
 
-#: src/widgets/preferences.rs:305
+#: src/widgets/preferences.rs:423
 msgid "Hex-Code"
 msgstr "Código hexadecimal"
 
-#: src/window.rs:460
+#: src/widgets/preferences.rs:429
+msgid "RGB"
+msgstr "RGB"
+
+#: src/widgets/preferences.rs:435
+msgid "HSL"
+msgstr "HSL"
+
+#: src/widgets/preferences.rs:443
+msgid "HSV"
+msgstr "HSV"
+
+#: src/widgets/preferences.rs:452
+msgid "CMYK"
+msgstr "CMYK"
+
+#: src/widgets/preferences.rs:461
+msgid "XYZ"
+msgstr "XYZ"
+
+#: src/widgets/preferences.rs:470
+msgid "CIELAB"
+msgstr "CIELAB"
+
+#: src/widgets/preferences.rs:482
+msgid "HWB"
+msgstr "HWB"
+
+#: src/widgets/preferences.rs:496
+msgid "CIELCh / HCL"
+msgstr "CIELCh / HCL"
+
+#: src/widgets/preferences.rs:503
+msgid "Name"
+msgstr "Nombre"
+
+#: src/window.rs:349 src/window.rs:635
+msgctxt "Information that no name for the color could be found"
+msgid "Not named"
+msgstr "Sin nombre"
+
+#: src/window.rs:537
+msgid "Failed to get palette color"
+msgstr "No se ha podido obtener el color de la paleta"
+
+#: src/window.rs:557
 msgid "Failed to pick a color"
 msgstr "No se ha podido elegir un color"
 
 #. Translators: Do not replace the {}. These are used as placeholders for the copied values
-#: src/window.rs:567 src/window.rs:574
+#: src/window.rs:655
```

Comment:
Are you sure, that you are on the latest commit? These line numbers don't seem quite right, they don't match up with the template file.

Reply:
What I did was to take the new, recently updated `de.po` file and remove the German strings and replace them with Spanish and Basque. But when making the pull request for some reason the previous strings were mixed with the ones I had updated. Maybe that's why.

Reply:
It's ok, worst case, I would have just fixed them myself.